packages/babel-plugin/src/utils/evaluate-path.js

```diff
@@ -690,6 +690,9 @@ function _evaluate(path: NodePath<>, state: State): any {
       ) {
         const val: number | string = object.node.value;
         func = (val as $FlowFixMe)[property.node.name];
+        if (typeof val === 'string') {
+          context = object.node.value;
+        }
```

Comment:
The previous logic. we only get the String's method. but we don't setup the context for the function call.If we don't apply the context for the function. It'll cause panic. But 
the Numeric's method isn't a valid expression. So i think judge string is enough.